CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Add Delete device button in Device Info Card ([#397](https://github.com/astarte-platform/astarte-dashboard/issues/397)).
 - Add Deletion in progress column for every device ([#398](https://github.com/astarte-platform/astarte-dashboard/issues/398)).
 - Report the device registration limit, how many devices can be registered, and why new devices cannot be registered ([#407](https://github.com/astarte-platform/astarte-dashboard/issues/407), [#408](https://github.com/astarte-platform/astarte-dashboard/issues/408)). 
+- Compute permissions from the JWT and disable unavailable UI sections, ([#416](https://github.com/astarte-platform/astarte-dashboard/issues/416)).
```

Comment:
Lines of changes cannot be added to this section since `v1.2.0-rc.0` was already released on 2024-05-28.

I suggest you begin the PR with a commit that specifically `Prepare next development cycle` by:
- bumping the Dashboard version from `1.2.0-rc.0` to `1.3.0-dev` in package.json and package-lock.json
- adding a new section here in the Changelog with just the title:
```
## [1.3.0-dev] - Unreleased
```

A different commit will add your line of changes under the new section, also adding the `### Added` subtitle.